server/ai_mediaserver.go

```diff
@@ -454,6 +455,10 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 					clog.Errorf(ctx, "failed to kick input connection: %s", kickErr.Error())
 				}
 				clog.Errorf(ctx, "Live AI auth failed: %s", err.Error())
+				monitor.SendQueueEventAsync("ai_stream_events", map[string]string{
+					"type":    "error",
+					"message": "Live AI auth failed: " + err.Error(),
```

Comment:
@ecmulli @gioelecerati wanted to check this would not break anything downstream since we only have a limited set of fields here. Most of the rest are unknown at this point - they come as part of the auth webhook, but the auth webhook has failed here.